api/bin/chainflip-lp-api/src/main.rs

```diff
@@ -619,6 +625,20 @@ pub struct LPOptions {
 		help = "A path to a file that contains the LP's secret key for signing extrinsics."
 	)]
 	pub signing_key_file: PathBuf,
+	#[clap(
+		id = "health_check.hostname",
+		long = "health_check.hostname",
+		help = "Hostname for this LP server's healthcheck. Requires --health_check.port to be present as well.",
+		requires("health_check.port")
+	)]
+	pub health_check_hostname: Option<String>,
+	#[clap(
+		id = "health_check.port",
+		long = "health_check.port",
+		help = "Port for this LP server's healthcheck. Requires --health_check.hostname to be present as well.",
+		requires("health_check.hostname")
+	)]
+	pub health_check_port: Option<u16>,
```

Comment:
Seems like we could dedupe this (would just require a more general log message): https://docs.rs/clap/3.2.16/clap/trait.Args.html#example

Reply:
Done.